helm/values.yaml

```diff
@@ -90,13 +87,17 @@ datatracker:
     #   cpu: 100m
     #   memory: 128Mi
 
```

Comment:
I don't think it's defined in the deployment template?

Reply:
Isn't that https://github.com/ietf-tools/datatracker/blob/145c9917252218e83639e2acbb2b444425b89a7a/helm/templates/deployments/datatracker.yaml#L27-L28 ?

Reply:
Ah indeed. I missed the different naming.